history/titles/00_k_stormwind.txt

```diff
@@ -418,6 +424,9 @@ d_moonbrook = {
 	598.3.17={
 		holder=0 #Edwin VanCleef
 	}
+	611.1.13={
+		holder= 5812 # Vanessa VanCleef
+	}
```

Comment:
Edwin hasn't the Moonbrook title because he has the Defias Brotherhood.

Reply:
Fixed in a73fedcf459b5b9f6d7920599cfffd5b7949cfe3.

Removed `d_moonbrook` from Vanessa as Edwin didn't have it either.